apis/vshn/v1/dbaas_vshn_postgresql.go

```diff
@@ -32,6 +33,9 @@ type VSHNPostgreSQL struct {
 type VSHNPostgreSQLSpec struct {
 	// Parameters are the configurable fields of a VSHNPostgreSQL.
 	Parameters VSHNPostgreSQLParameters `json:"parameters,omitempty"`
+
+	// WriteConnectionSecretToRef references a secret to which the connection details will be written.
+	WriteConnectionSecretToRef corev1.LocalObjectReference `json:"writeConnectionSecretToRef,omitempty"`
```

Comment:
This is the actual change I need right now. This allows me to access the connection secret for the SLI exporter